registry/accessing-the-registry.adoc

```diff
@@ -49,8 +49,10 @@ include::modules/registry-viewing-logs.adoc[leveloffset=+1]
 
 include::modules/registry-accessing-metrics.adoc[leveloffset=+1]
 
-.Additional resources
+include::modules/
 
+== Additional resources
+* For more information on allowing pods to reference images across projects, see xref:../openshift_images/managing_images/using-image-pull-secrets.adoc#images-allow-pods-to-reference-images-across-projects_using-image-pull-secrets[Allowing pods to reference images across projects].
```

Comment:
You could vary the description leading up to the cross-reference from the cross-reference heading, to prevent repetition. For example:

`* For more information on allowing pods in a project to reference images in another project, see xref:../openshift_images/managing_images/using-image-pull-secrets.adoc#images-allow-pods-to-reference-images-across-projects_using-image-pull-secrets[Allowing pods to reference images across projects].`